go/vt/sqlparser/comments.go

```diff
@@ -252,3 +260,32 @@ func (d CommentDirectives) IsSet(key string) bool {
 	}
 	return false
 }
+
+// SkipQueryPlanCacheDirective returns true if skip query plan cache directive is set to true in query.
+func SkipQueryPlanCacheDirective(stmt Statement) (skipQuerPlanCacheDirective bool) {
```

Comment:
`skipQuerPlanCacheDirective` has a typo :)

Also IMO the named return var is really long here. I'd shorten to `skip` or just omit it altogether and have a local var which defaults to false.